docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1072,212 +1072,621 @@ thor:
-              "compile-." or exact match "^compile-.$"</para>
-            </listitem>
+            <colspec />
 
-            <listitem>
-              <para>All: to apply for all HPCC Systems components. The default
-              placements for pods we deliver is [all]</para>
-            </listitem>
-          </orderedlist>
-
-          <para><emphasis role="bold">Placements</emphasis> – in Kubernetes
-          the Placement concept allows you to spread your pods across types of
-          nodes with particular characteristics. Placements would be used to
-          ensure that pods or jobs that want nodes with specific
-          characteristics are placed on them.</para>
-
-          <para>For instance a Thor cluster could be targeted for machine
-          learning using nodes with a GPU. Another job may want nodes with a
-          good amount more memory or another for more CPU. You can use
-          placements to ensure that pods with specific requirements are placed
-          on appropriate nodes.</para>
+            <tbody>
+              <row>
+                <entry>Type: &lt;component&gt;</entry>
+
+                <entry>Covers all pods/jobs under this type of component. This
+                is commonly used for HPCC Systems components. For example, the
+                <emphasis>type:thor</emphasis> which will apply to any of the
+                Thor type components; thoragent, thormanager, thoragent and
+                thorworker, etc.</entry>
+              </row>
+
+              <row>
+                <entry>Target: &lt;name&gt;</entry>
+
+                <entry>The "name" field of each component, typical usage for
+                HPCC Systems components referrs to the cluster name. For
+                example <emphasis>Roxie: -name: roxie</emphasis> which will be
+                the "Roxie" target (cluster). You can also define multiple
+                targets with each having a unique name such as "roxie",
+                "roxie2", "roxie-web" etc.</entry>
+              </row>
+
+              <row>
+                <entry>Pod: &lt;name&gt;</entry>
+
+                <entry>This is the "Deployment" metadata name from the name of
+                the array item of a type. For example, "eclwatch-", "mydali-",
+                "thor-thoragent" This can be a regular expression since
+                Kubernetes will use the metadata name as a prefix and
+                dynamically generate the pod name such as,
+                eclwatch-7f4dd4dd44cb-c0w3x.</entry>
+              </row>
+
+              <row>
+                <entry>Job name:</entry>
+
+                <entry>The job name is typically a regular expression as well,
+                since the job name is generated dynamically. For example, a
+                compile job compile-54eB67e567e, could use "compile-" or
+                "compile-*" or the exact match "^compile-.$"</entry>
```

Comment:
@xwang2713 - I want to clarify if this supports regular expressions or just wildcards, "compile-*" is not an example of a regular expression (well it is, but would match 'compile----' , or 'compile', not 'compile-blah')

Reply:
Either "compile-.*" or "compile-".  "compile-*" is not valid regular expression or substring

Reply:
Ok, then line 1308 needs changing to "compile-.\*"
@g-pan